app/services/import/legacy/fetch-company.service.js

```diff
@@ -29,24 +29,6 @@ function _query () {
   return `
   SELECT DISTINCT ON (np."ID")
     (concat_ws(':', np."FGAC_REGION_CODE", np."ID")) AS external_id,
-    (
-      CASE np."SALUTATION"
-        WHEN 'null' THEN NULL
-        ELSE np."SALUTATION"
-        END
-      )  AS salutation,
-    (
-      CASE np."FORENAME"
-        WHEN 'null' THEN NULL
-        ELSE np."FORENAME"
-        END
-      )  AS firstName,
-    (
-      CASE np."NAME"
-        WHEN 'null' THEN NULL
-        ELSE np."NAME"
-        END
-      )  AS lastName,
```

Comment:
Check this trim is actullay needed

Reply:
I think this is needed. 

This was the previous code: 

```
const mapName = party => {
  const firstNameKey = party.FORENAME === 'null' ? 'INITIALS' : 'FORENAME'

  const parts = [
    party.SALUTATION,
    party[firstNameKey],
    party.NAME
  ]

  return parts
    .map(str.mapNull)
    .filter(value => value)
    .join(' ')
}
```

We can see in the database there are companies (parties) where the `APAR_TYPE = person` and in WRLS the name in the company table is the result of this logic.